hedera-node/hedera-file-service-impl/src/test/java/com/hedera/node/app/service/file/impl/test/handlers/FileUpdateTest.java

```diff
@@ -148,6 +154,27 @@ void appliesNewKeys() {
         assertEquals(expectedKey, newFile.keys());
     }
 
+    @Test
+    @DisplayName("Fails handle if keys doesn't exist on file to be updated")
+    void keysDoesntExist() {
```

Comment:
Might be worth renaming to "failForImmutableFile"

Reply:
done thanks

Reply:
done thanks